src/pip/_internal/cli/base_command.py

```diff
@@ -12,10 +12,7 @@
 
 from pip._internal.cli import cmdoptions
 from pip._internal.cli.command_context import CommandContextMixIn
-from pip._internal.cli.parser import (
-    ConfigOptionParser,
-    UpdatingDefaultsHelpFormatter,
-)
+from pip._internal.cli.parser import ConfigOptionParser, UpdatingDefaultsHelpFormatter
```

Comment:
@pradyunsg you argued that putting imports on one line is bad for git diff / conflicts (https://github.com/pypa/pip/pull/8902#issuecomment-697503571). How is this better?

Reply:
The rule here is "break out imports when they go over the line limit". This is *exactly* the rule we have had since we adopted isort. We obviously don't want things like:

```
from os import (
    environ,
    path,
)
```

---

The comment you reference is about what to do *after* we've broken out the import across multiple lines and I don't think anyone has ever argued we should have multiple lines if things fit in a single line.